src/osd/OSDMap.cc

```diff
@@ -5216,6 +5223,19 @@ void OSDMap::rm_all_upmap_prims(CephContext *cct, OSDMap::Incremental *pending_i
   }
 }
 
+void OSDMap::rm_all_upmap_prims(CephContext *cct, OSDMap::Incremental *pending_inc) {
```

Comment:
nit: bracing.

Reply:
ACK. Fixed the bracing in the .h file as well.